.editorconfig

```diff
@@ -1,6 +1,69 @@
 # top-most EditorConfig file
 root = true
 
-[*.cs]
+[*.{cs,vb}]
 indent_style = space
-indent_size = 4
\ No newline at end of file
+indent_size = 4
+
+# Organize usings
+dotnet_sort_system_directives_first = true 
+
+# this. preferences
+dotnet_style_qualification_for_field = false:silent
+dotnet_style_qualification_for_property = false:silent
+dotnet_style_qualification_for_method = false:silent
+dotnet_style_qualification_for_event = false:silent
+
+# Language keywords vs BCL types preferences
+dotnet_style_predefined_type_for_locals_parameters_members = true:suggestion
+dotnet_style_predefined_type_for_member_access = true:suggestion
+
+# Parentheses preferences 
+dotnet_style_parentheses_in_arithmetic_binary_operators = always_for_clarity:silent 
+dotnet_style_parentheses_in_relational_binary_operators = always_for_clarity:silent 
+dotnet_style_parentheses_in_other_binary_operators = always_for_clarity:silent 
+dotnet_style_parentheses_in_other_operators = never_if_unnecessary:silent
```

Comment:
This is the default. I'm not completely sure how it works out in all the cases and combinations of operators, but it should never require anything since it is silent. The last option is for simple cases like `var x = (value)`.